docs/docs/getting-started/web.mdx

```diff
@@ -34,7 +35,7 @@ Consider adding it to your `postinstall` script.
 
 Once you are done, you need to pick your strategy to [Load Skia](#loading-skia).
 
-If you are [loading CanvasKit from a CDN](#using-a-cdn), you don't need to run the `setup-skia-web` script. If you are not using `react-native-reanimated`, webpack will output a warning because React Native Skia refers to Reanimated. We describe how to fix this warning [here](#manual-webpack-installation).
+If you are [loading CanvasKit from a CDN](#using-a-cdn), you don't need to run the `setup-skia-web` script for webpack. If you are not using `react-native-reanimated`, webpack will output a warning because React Native Skia refers to Reanimated. We describe how to fix this warning [here](#manual-webpack-installation).
```

Comment:
This should to be updated

Reply:
I don't have all the historic context with webpack here, not sure how you want to update this line.